.devcontainer/scripts/post_create_command.sh

```
@@ -0,0 +1,15 @@
+
+echo "Updating package list..."
+sudo apt update
+rosdep update
+
+echo "Installing ROS packages..."
+make custom-deps deps
+
+echo "Sourcing ROS workspace..."
+echo "source /opt/ws/install/setup.bash" >> ~/.bashrc
+echo "source /opt/ws/.devcontainer/openmower_config.env" >> ~/.bashrc
+source ~/.bashrc
```

Shellcheck

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)